config.yml

```diff
@@ -779,6 +909,266 @@ production_factories:
     repair_inputs:
       Diamond:
         material: DIAMOND
+  Random_Wooden_Tool_Factory:

```

Comment:
It might be "too late" but you should never have rebased/merged against ribagi's changes while formulating your own. Regardless of XP rework, the tool factory stuff is going in; frankly they should have been separate pulls but tl;dr they aren't.

In any case, it's going to introduce merge errors and regressions to have them here. If you can, I'd recommend checking out master and layering *strictly speaking* just your XP proposal against master config, then opening a new pull.